Remotion/Web/Core/UI/Controls/IValidatorWithDynamicErrorMessage.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.Web.UI;
+
+namespace Remotion.Web.UI.Controls
+{
+  public interface IValidatorWithDynamicErrorMessage: IValidator
+  {
+    /// <summary>Performs validation on the already validated associated input control again and updates the <see cref="P:System.Web.UI.WebControls.BaseValidator.IsValid" />
+    /// property if there are no more validation failures.</summary>
```

Comment:
Message needs updating since we no longer validation but only update the error message.